guides/_subsections/store-credentials.md

```diff
@@ -0,0 +1,47 @@
+
+<ol>
+  <li>
+    {% include howto_step.html
+      summary="Navigate to the Security tab"
+      image-name="2.3.0/navigate-to-security.png"
+      text="
+        First, navigate to the <i>Security</i> tab. The Store Credentials table lists all credentials in Cryostat's keyring. Click <i>Add</i> to define a new set of credentials.
+      "
+    %}
+  </li>
+  <li>
+    {% include howto_step.html
+      summary="Enter your credentials"
+      image-name="2.3.0/store-credentials-1.png"
+      text="
+          A modal will appear. You can select a target JVM to view its properties.
+          These properties can be used to define a <i>matchExpression</i> that tests whether this set of credentials should apply to the selected target JVM.
+          This is the same kind of <i>matchExpression</i> as those used by <a href='#create-an-automated-rule'>Automated Rules</a>, so you may find it useful to reuse the same
+          <i>matchExpression</i> from your rule definitions for your credentials definitions.
+          The <i>matchExpression</i>, username, and password fields are all required.
+      "
+    %}
+  </li>
+    <li>
+    {% include howto_step.html
+      summary="(Alternative to Steps 1 and 2) Store Credentials when connecting to a target JVM"
+      image-name="2.3.0/navigate-to-recordings-1.png"
+      text="
+        Alternatively, credentials may also be stored if you navigate to either the <i>Recordings</i> tab or the <i>Events</i> tab and select a target JVM with authentication enabled. The authentication form will appear, prompting you to enter your credentials. If you have <a href='#configure-credentials-storage'>configured Backend Credentials Storage</a>, then a new credentials definition with the provided username and password will be stored for this specific target application in the Cryostat keyring. If you have configured credentials with the <i>Session</i> option then these entered credentials will <i>not</i> be stored in the Cryostat keyring.
+      "
+    %}
+  </li>
+  <li>
+    {% include howto_step.html
+      summary="View your stored credentials"
+      image-name="2.3.0/store-credentials-2.png"
+      text="
+        The <i>Store Credentials</i> table will update with rows for each of the credentials you have stored in the keyring. The <i>matchExpression</i> is visible along with a count of the number of known target applications that these credentials will apply to. You can click the reverse chevron icon at the left of the table row to expand the row, which will display a list of those matched target applications. For security reasons it is not possible to view the stored username and password associated with a credentials definition in the keyring.
```

Comment:
> The <i>Store Credentials</i> table will update with rows for each of the credentials

Seems it suggests each credential will have multiple rows representing it.

Also:

> You can click the reverse chevron icon at the left of the table row to expand the row, which will display a list of those matched target applications.

This might not be true in small viewport (maybe mobile phone?).

![Screenshot from 2023-04-27 15-55-38](https://user-images.githubusercontent.com/68053619/234976644-93c2f2f4-11e0-4921-8bcf-aaf3bc6ca177.png)